src/pages/jeongilLee/Main/Main.js

```diff
@@ -1,13 +1,342 @@
-import React, { Component } from 'react';
+/* eslint-disable*/
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Main.scss';
+import Comment from './Comment';
+
+class Main extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      id: 1,
+      commentInfo: [],
+      userId: 'test',
+      reply: '',
+      input: React.createRef(),
+      className: 'commentSubmit',
+    };
+  }
+
+  newComment = e => {
+    this.setState({
+      reply: e.target.value,
+    });
+  };
+
+  pressEnter = e => {
+    if (e.key === 'Enter') this.submitReply();
+  };
+
+  submitReply = () => {
+    const { id, userId, reply, commentInfo } = this.state;
+
+    // const obj = {
+    //   id,
+    //   userId,
+    //   reply,
+    // };
+
+    // const newComments = commentInfo.concat(obj);
+    const newComments = [...commentInfo, { id, userId, reply }];
+
+    this.setState({
+      id: id + 1,
+      commentInfo: newComments,
+      reply: '',
+    });
```

Comment:
```suggestion
    this.setState({
      id: id + 1,
      commentInfo: [...commentInfo, { id, userId, reply }],
      reply: '',
    });
```
전개연산자 사용 너무 좋습니다 :)!!!

Reply:
이렇게 더 간단히 바꿀 수도 있겠네요 :)